src/containers/layout/footer/Footer.styles.ts

```diff
@@ -6,14 +6,20 @@ export const styles = {
   },
   container: {
     display: 'flex',
+    position: 'relative',
     flexDirection: { xs: 'column-reverse', sm: 'row' },
     justifyContent: { xs: 'space-between', sm: 'space-around' },
     alignItems: 'center',
     py: { xs: '12px', sm: '26px' }
   },
+  title: {
+    position: { xs: 'static', sm: 'absolute' },
+    left: { xs: '0', sm: '0' }
+  },
   links: {
     display: { xs: 'flex', sm: 'block' },
     flexDirection: { xs: 'row', sm: 'column' },
+    marginLeft: { xs: '0', sm: '48px' },
```

Comment:
why do we need a margin if we need them to be in center

Reply:
becouse of overlaping

![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/96788072/c9d58258-f24d-47ff-b02d-4b6963a625bb)